src/test/kotlin/com/expedia/graphql/KotlinSchemaDirectiveWiringHelperTest.kt

```diff
@@ -0,0 +1,246 @@
+// package com.expedia.graphql
```

Comment:
Is this WIP?

Reply:
Yes, this PR is in progress. Any feedback about the architecture changes is appreciated. We could have used the draft pr feature but this is fine